lib/hls/manifest_text_parser.js

```diff
@@ -158,7 +158,13 @@ shaka.hls.ManifestTextParser = class {
             partialSegmentTags.push(tag);
           } else if (tag.name == 'EXT-X-PRELOAD-HINT') {
             if (tag.getAttributeValue('TYPE') == 'PART') {
-              partialSegmentTags.push(tag);
+              if (tag.getAttributeValue('BYTERANGE-START') != null) {
```

Comment:
Please add a comment here that BYTERANGE-START without BYTERANGE-LENGTH is being ignored.

Reply:
Done!